vulyk/blueprints/gamification/__init__.py

```diff
@@ -35,6 +37,12 @@ def __init__(self, *args, **kwargs):
             k: 1 if k == 1 else (k - 1) * 25 for k in range(1, 51)
         }
 
+    def register(self, app, options, first_registration=False):
+        super().register(app, options, first_registration)
+
+        if app.config.get('ENABLE_ADMIN', False):
+            app.admin.add_view(AuthModelView(FundModel))
```

Comment:
Could `app.admin` by any chance happen to be uninitialized here?
It'd be better to add some error handling in here with proper exception being wrapped and re-thrown with meaningful message.

Also, why don't you bind `RuleModel` to admin panel?